test/lib/tasks/upgrades/4.1/sync_noarch_content_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'katello_test_helper'
+require 'rake'
+
+module Katello
+  class SyncNoarchContentTest < ActiveSupport::TestCase
+    def setup
+      Rake.application.rake_require 'katello/tasks/upgrades/4.1/sync_noarch_content'
+      Rake::Task['katello:upgrades:4.1:sync_noarch_content'].reenable
+      Rake::Task.define_task(:environment)
+    end
+
+    def test_sync_content_with_noarch_repository
+      count = Katello::RootRepository.where(arch: 'noarch').count
```

Comment:
i'm worried that the test fixtures could change one day and maybe count would be 0. Does the `.times` method throw an error for 0? If not , might be worth adding an assertion to make sure we actually are hitting the Content.update method at least once